app/src/main/res/values/colors.xml

```diff
@@ -59,6 +59,8 @@
     <color name="base10">#eaeaea</color>
     <color name="base30">#c1c1c1</color>
 
+    <color name="mtrl_textinput_default_box_stroke_color" tools:override="true">#fff</color>
```

Comment:
hey, i don't think this is a good practice here because we don't want to set the default `boxStrokeColor` of `EditText` to white as it is hardly used in any other activity. The default `boxStrokeColor` should be `#4C56C0` as it is mostly used.